src/record/export/repositories/stringifiers/index.ts

```diff
@@ -0,0 +1,21 @@
+import type { RecordSchema } from "../../types/schema";
+
+import { CsvStringifier } from "./csvStringifier";
+import type { Duplex } from "stream";
+
+export type Stringifier = Duplex;
```

Comment:
As mentioned in 'LocalRecordRepository', `Duplex` has too weak type as an interface.  The `Stringifier` provides such interface:
- An user can write LocalRecord(s).
- An user can notify the end of writing.
- An user can write stringified result to the stream.

I prefer such interface:

```typescript
export interface Stringifier {
  write(records: LocalRecord[]): void;
  end(): void;

  pipe(w: NodeJS.WritableStream): void;
}
```